config/routes.rb

```diff
@@ -17,7 +17,6 @@
     resources :deploys, only: [:index, :show, :destroy] do
       collection do
         get :active
-        get :active_count
```

Comment:
nope, no per-project active counter

On Tue, May 17, 2016 at 9:59 AM, Shane Hender <notifications@github.com>
wrote:

> In config/routes.rb
> <https://github.com/zendesk/samson/pull/985#discussion_r63561643>:
>
> > @@ -17,7 +17,6 @@
> >      resources :deploys, only: [:index, :show, :destroy] do
> >        collection do
> >          get :active
> > -        get :active_count
>
> Need this line back right?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/samson/pull/985/files/7139a07ce7500a1ebd185bf9c328eb1cb443dffb#r63561643>
>